src/tests/ftest/util/job_manager_utils.py

```diff
@@ -5,7 +5,7 @@
 """
 # pylint: disable=too-many-lines
 from datetime import datetime
-from distutils.spawn import find_executable
+from distutils.spawn import find_executable  # pylint: disable=deprecated-module
```

Comment:
This is a new warning, and I think we use it in the build as well, we should file a ticket to remove this.

Reply:
https://daosio.atlassian.net/browse/DAOS-13130